Scripts/summaryStatistics/biophysics/density_calculation.py

```diff
@@ -221,8 +221,7 @@ def main(cells, fig_export_path=''):
     bw_img = draw_image_bw(img_dimensions, cell_centers_x, cell_centers_y, cell_lengths, cell_radii, cell_orientations)
```

Comment:
for the "draw_cell" function which is called by the "draw_image_bw" function: instead of calculating the following lines 4 times, you can calculate them once by defining a variable for each sum and calling the variables (each) 2 times. the lines are as below:
 p1 = (center_x + half_length_along_axis_x + radius_perpendicular_to_axis_x, center_y + half_length_along_axis_y + radius_perpendicular_to_axis_y)
    p2 = (center_x + half_length_along_axis_x - radius_perpendicular_to_axis_x, center_y + half_length_along_axis_y - radius_perpendicular_to_axis_y)
    p3 = (center_x - half_length_along_axis_x - radius_perpendicular_to_axis_x, center_y - half_length_along_axis_y - radius_perpendicular_to_axis_y)
    p4 = (center_x - half_length_along_axis_x + radius_perpendicular_to_axis_x, center_y - half_length_along_axis_y + radius_perpendicular_to_axis_y)